pkg/users/account-create-dialog.js

```diff
@@ -125,6 +140,10 @@ function validate_real_name(real_name) {
         return _("The full name must not contain colons.");
 }
 
+function validate_home_dir(dir) {
+    return cockpit.spawn(["[", "!", "-d", dir, "]"], { superuser: "require" });
```

Comment:
This is more or less shell syntax. Can you please do `test ! -d dir` instead? That's a bit easier to read.